### PR TITLE
Fix container_description workflow

### PR DESCRIPTION
**Pull request by SuperQ**
Fix yaml indentation. :facepalm:

### CODE REVIEW

File: .github/workflows/container_description.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
     paths:
       - "README.md"
       - ".github/workflows/container_description.yml"
-  branches: [ main, master ]
```
Comment (machine424) [inline .github/workflows/container_description.yml:8]: oops. Maybe we can think about adding a linter for such workflows that don't run on each PR...
Maybe we could take advantage of this to adjust the name + some of the steps names under `PushQuayIoReadme` (they're still mentioning docker hub)

Comment (SuperQ) [reply .github/workflows/container_description.yml:8]: Yea, probably a good TODO item. 

Fixed the step names.

Comment (machine424) [reply .github/workflows/container_description.yml:8]: I'll create an issue.
A last one in line 35, but we can keep it as a "first contribution" to someone else ;)

**Review by machine424 — APPROVED** [review]
lgtm

**Pull request merged**